en-US/New.xml

```diff
@@ -5,6 +5,66 @@
 ]>
 <section id="new">
         <title>New in This Release</title>
+		<formalpara id="new_62">
+		<title>Release 6.2</title>
+            <para>Update in February 2024 to provide added or updated guidance and to address reported issues.</para>
+                </formalpara>   
+            <itemizedlist>
+				<listitem>
+                                        <para>Added guidance about line continuation for multiple operating systems.
+										Refer to <xref linkend="continuation"/>.</para>			
+				</listitem>
+				<listitem>
+                                        <para>Various updates to guidance about punctuation, including in lists and in tables.
+										Refer to <xref linkend="punctuation"/>.</para>			
+				</listitem>
+				<listitem>
+                                        <para>Updated guidance about referring to special characters.
+										Refer to <xref linkend="referring"/>.</para>			
+				</listitem>
+				<listitem>
+                                        <para>You can use the possessive form of the company name (Red&nbsp;Hat's) to refer to the company name itself, not to any products or services.
```

Comment:
I'm not quite following this sentence. What does it mean to use a possessive to refer to the company name itself? (It seems to be suggesting something like "Red Hat's Red Hat".) By definition, the possessive needs to refer to something owned by the company, such as product or service, which contradicts the first part of the sentence. Can this statement be clarified?